lib/V8/v8-globals.cpp

```diff
@@ -217,30 +217,30 @@ TRI_v8_global_t::SharedPtrPersistent::emplace(
 
 TRI_v8_global_t::~TRI_v8_global_t() = default;
 
-v8::Local<v8::Object> TRI_GetObject(v8::Local<v8::Context>& context,
+v8::Local<v8::Object> TRI_GetObject(v8::Local<v8::Context> context,
```

Comment:
This looks weird; is the pass by reference a historic thing?

Reply:
Normally it is good to pass things by reference to avoid copies.
All the v8 objects however can (and should) be passed by value, as they are just nice wrappers around pointers.
So moving from by-reference to by-value here was intentional and should not make things any worse.